cocos/gfx/webgl/webgl-swapchain.ts

```diff
@@ -245,12 +245,12 @@ export class WebGLSwapchain extends Swapchain {
             WebGLDeviceManager.instance.capabilities.maxTextureUnits,
             WebGLDeviceManager.instance.capabilities.maxVertexAttributes,
         );
-
+        this._format = premultipliedAlpha ? Format.RGBA8 : Format.RGB8;
```

Comment:
Why not use RGBA8 for both cases?
`premultiplied alpha` should not have affected pixel format.

Reply:
When premultipliedAlpha is set to false, the format of the swapchain is RGB8, not RGBA8. You can obtain this through gl.drawingBufferFormat.

Reply:
I see.